doc/release-process.md

```diff
@@ -50,15 +50,15 @@ and sort them into categories based on labels)
 
 Generate list of authors:
 
-    git log --format='%aN' "$*" | sort -ui | sed -e 's/^/- /'
+    git log --format='- %aN' v(current version, e.g. 0.16.0)..v(new version, e.g. 0.16.1) | sort -ui
```

Comment:
nit: Could pass `--ignore-case` to sort

Reply:
@fanquake right!

Interestingly I was using the `sort` utility on Ubuntu 16.04 and it strangely ignores case by default, so I didn't spot this issue. However, I just tested this on my Mac using both `sort` and `gsort` and it definitely requires this flag.

I have now updated the commit with the changes.